packages/graphback-core/src/runtime/QueryFilter.ts

```diff
@@ -29,7 +29,11 @@ export const FILTER_SUPPORTED_SCALARS = [
   'GraphbackTimestamp',
   'GraphbackTime',
   'GraphbackDate',
-  'GraphbackDateTime'
+  'GraphbackDateTime',
+  'Timestamp',
+  'Time',
+  'Date',
+  'DateTime'
```

Comment:
Good quick fix. 

It looks like we do not generate filters for all the custom scalars, but known ones (as in 0.15 release). Maybe we should add it as breaking change and mention it in the filters docs as a `caution`. What do you think?

Reply:
Yeah, I am on board with that!

Reply:
Thank you :-)

Reply:
How is this? https://github.com/aerogear/graphback/pull/1906/commits/fff7a323c681f04d72b6c689a7fc436474f46ac3

Reply:
Great. I added a small suggestion. 

Also with https://github.com/aerogear/graphback/pull/1870 we disable support for unknown scalars. What do you think about adding it to the release note?

Reply:
Thanks! Added now in https://github.com/aerogear/graphback/pull/1906/commits/2358b6ad0d522a1e575c98dedc5fa8d0069c3188